docs/about/join.md

```diff
@@ -92,13 +108,12 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
     <input type="checkbox" id="ucsf_compute_environments_2" name="ucsf_compute_environments" value="qb3"/> QB3 (now deprecated)<br>
     <input type="checkbox" id="ucsf_compute_environments_3" name="ucsf_compute_environments" value="radiology"/> Radiology<br>
     <input type="checkbox" id="ucsf_compute_environments_4" name="ucsf_compute_environments" value="sali"/> Sali Lab<br>
-    <input type="checkbox" id="ucsf_compute_environments_5" name="ucsf_compute_environments" value="tipcc"/> TIPCC (Cancer Center compute cluster)<br>
```

Comment:
Why was TIPCC dropped?  Mistake?

Reply:
@HenrikBengtsson Since TIPCC didn't join Wynton, the UID and GID provided will be not relevant. It was intentional.

Reply:
Moving this over to Gitea.